package.json

```diff
@@ -42,34 +43,40 @@
     "npm": ">=3.0.0"
   },
   "devDependencies": {
-    "aegir": "^18.2.1",
+    "aegir": "^19.0.3",
     "chai": "^4.2.0",
     "dirty-chai": "^2.0.1",
     "lodash": "^4.17.11",
-    "memdown": "^3.0.0",
+    "memdown": "^4.0.0",
     "multihashes": "~0.4.14",
-    "multihashing-async": "~0.5.2",
+    "multihashing-async": "~0.7.0",
     "ncp": "^2.0.0",
     "rimraf": "^2.6.3"
   },
   "dependencies": {
-    "async": "^2.6.2",
     "base32.js": "~0.1.0",
-    "bignumber.js": "^8.1.1",
+    "bignumber.js": "^9.0.0",
     "buffer": "^5.2.1",
     "cids": "~0.7.0",
-    "datastore-core": "~0.6.0",
-    "datastore-fs": "~0.8.0",
-    "datastore-level": "~0.11.0",
+    "datastore-core": "~0.7.0",
+    "datastore-fs": "~0.9.0",
+    "datastore-level": "~0.12.0",
     "debug": "^4.1.0",
     "dlv": "^1.1.2",
-    "interface-datastore": "~0.6.0",
+    "err-code": "^1.1.2",
+    "interface-datastore": "~0.7.0",
     "ipfs-block": "~0.8.1",
     "just-safe-set": "^2.1.0",
+    "lodash.get": "^4.4.2",
+    "lodash.has": "^4.5.2",
+    "lodash.set": "^4.3.2",
```

Comment:
Use `just-safe-set` and `just-safe-get` for smaller bundle size?

Reply:
The bundled produced is 20kb smaller with `just-safe-set` and `just-safe-get` instead of `lodash.get` and `lodash.set`.